README.md

```diff
@@ -165,6 +165,32 @@ where they will briefly update the limits. After you get confirmation that they
 
 ---
 
+# AltLayer
+
+## Operator Flow
+
+1. generate and encrypt a new BLS keystore using the EigenLayer CLI tooling https://docs.eigenlayer.xyz/eigenlayer/operator-guides/operator-installation#create-and-list-keys
```

Comment:
It looks like you are missing an additional step mentioned in the altlayer operator guide

It looks like they must also
1. generate an additional ECDSA key that is the `node` address
2. Fund that key with at least 0.3 eth
3. fill out some form

then later down in that registration guide it says the operator must `opt-in`. You need to verify whether this operation can be run with just that `node` key from above or if this is another command/call that will need to be run by the ether.fi admin

Reply:
Verified with the altlayer team that `opt-in` just calls `registerOperator` under the hood and that the node operators don't need the ECSDA private key to run the AVS